pkg/proxy/node_test.go

```diff
@@ -17,13 +17,21 @@ limitations under the License.
 package proxy
 
 import (
+	"k8s.io/klog/v2"
+	"strconv"
 	"testing"
 
 	v1 "k8s.io/api/core/v1"
```

Comment:
This import should be here.
```suggestion
	v1 "k8s.io/api/core/v1"
```